charts/ambassador/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 ## Next Release
 
+(no changes yet)
+
+## v6.7.11
+
+- Update Ambassador API Gateway chart image to version v1.13.8: [CHANGELOG](https://github.com/emissary-ingress/emissary/blob/master/CHANGELOG.md)
+- Update Ambassador Edge Stack chart image to version v1.13.8: [CHANGELOG](https://github.com/emissary-ingress/emissary/blob/master/CHANGELOG.md)
```

Comment:
Should we give this an `Identifier:` prefix of sorts?

Reply:
ah, probably. will add to release improvements